AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestRunnerService.kt

```diff
@@ -275,5 +281,10 @@ interface TestRunnerService {
          * XML result files produced by the test.
          */
         val xmlResults: List<ResultFileResource>
+
+        /**
+         * Test case log files produced by the test.
+         */
+        val testCaseLogcats: Map<String, ResultFileResource>
```

Comment:
as mentioned above, we likely want to avoid returning this here (as it will be big).
instead, query test by test when we are reporting it. Though i'm not sure how we'll associate test with log on the androidx.dev / ANts side. (so my comment my change based on that end)

Reply:
Leaving it as a map for now. I'll update this when we change the implementation to use the other FTL api: https://firebase.google.com/docs/test-lab/reference/toolresults/rest/v1beta3/projects.histories.executions.steps.testCases/list